.circleci/config.yml

```diff
@@ -151,13 +140,22 @@ workflows:
               only: /.*/
             branches:
               ignore: /^pull\/.*/
-      - deploy-docs:
+      # Deploy docs when commit to master
+      - deploy-docs-branch:
           requires:
-            - lint-and-unit-tests
-            - integration-tests
+          - build
```

Comment:
I still think the complete build i.e. lint/unit test and integration tests should pass before we deploy the docs.

Reply:
right, I'll put it back